packages/javascript/bh-shared-ui/src/components/AssetGroupFilters/AssetGroupFilters.tsx

```diff
@@ -27,7 +27,7 @@ import {
     Paper,
     Select,
 } from '@mui/material';
-import { Theme } from '@mui/material/styles';
+import { Theme, useTheme } from '@mui/material/styles';
```

Comment:
Im getting an unresolved dependency pointing to this import when building `bh-shared-ui`, in other places we import  `Theme` from`@mui/material`

Reply:
Good catch! This has been updated with pulling in #735